test/entrypoint/srv_ddl.lua

```diff
@@ -188,7 +191,9 @@ package.preload['customers-storage'] = function()
                 box.space['_ddl_sharding_key']:update(space_name, {{'=', fieldno_sharding_key, sharding_key_def}})
             end)
             rawset(_G, 'set_sharding_func', function(space_name, fieldno_sharding_func, sharding_func_def)
-                box.space['_ddl_sharding_func']:update(space_name, {{'=', fieldno_sharding_func, sharding_func_def}})
+                local record = {space_name}
+                record[fieldno_sharding_func] = sharding_func_def
+                box.space['_ddl_sharding_func']:upsert(record, {{'=', fieldno_sharding_func, sharding_func_def}})
```

Comment:
Why not replace? If there is a both second and third field will be filled, it would be an invalid ddl schema.

Reply:
Good idea, changed:

https://github.com/tarantool/crud/blob/1d98a493fda2765dd4a6ecb8d93a38bd9293282d/test/entrypoint/srv_ddl.lua#L196